Pmm 9452 change admin user password3 (#705)
percona/pmm-doc: docs/faq.md

```

(This example assumes your Docker container is named `pmm-server`.)

> ## How to change the PMM password for a deafult admin user?

Reviewer: ```suggestion
## How to change the PMM password for a default admin user?
```
deafult ->  default

> If you're deploying through Docker you can use change the password for an admin user after starting the Docker container as follows:

Reviewer: Maybe will be better:
```suggestion
If you're deploying through Docker you need to change the default password for an admin user after starting the Docker container as follows:
```

    * For PMM versions 2.27.0 and later:

    ```sh
    docker exec -t pmm-server change-admin-password <new_password>
    ```

    * For PMM versions prior to 2.27.0:

        ```sh
        docker exec -t pmm-server bash -c 'grafana-cli --homepath /usr/share/grafana --configOverrides cfg:default.paths.data=/srv/grafana admin reset-admin-password newpass'

>         ```

Reviewer: Why do we have other indents here?
